server/KubeJob.go

```diff
@@ -76,14 +92,10 @@ func GetJobStatus(w http.ResponseWriter, r *http.Request) {
 			res["LastProbeTime"] = strings.Replace(jobStatus.CompletionTime.String(), "T", " ", 1)[:20]
 		}
 
-		if jobStatus.Active == 0 &&
-			jobStatus.Failed == 0 &&
-			jobStatus.Succeeded != 0 {
+		if jobStatus.Active == 0 && jobStatus.Failed == 0 && jobStatus.Succeeded != 0 {
```

Comment:
This makes line length > 80. Hence I had to keep the previous fomat

Reply:
is that a go standard?
over time I have come to prefer this way as it makes it easier to read the logic. Only break is is longer than 3 conditions.

Reply:
It's not a go standard. I prefer keeping line lengths < 80 chars for readability